express-static/server.js

```diff
@@ -0,0 +1,9 @@
+import express from 'express';
+
+const app = express();
+
+app.use(express.static('public'));
+
+app.listen(8090, () => {
+  console.log('Express server listening on port 8080');
```

Comment:
Preventing mismatched port numbers is generally why it's a good idea to tie them to the same variable.